crates/uv/tests/pip_install.rs

```diff
@@ -855,7 +855,9 @@ fn install_extra_index_url_has_priority() {
         // find it, but then not find a compatible version. After
         // the fix, `uv` will check pypi.org first since it is given
         // priority via --extra-index-url.
-        .arg("black==24.2.0"), @r###"
+        .arg("black==24.2.0")
+        .arg("--exclude-newer")
+        .arg("2024-03-09"), @r###"
```

Comment:
I think @charliermarsh fixed this in a follow-up PR, but yeah, this test was very specifically _not_ using `--exclude-newer`. I think we also could have updated the snapshot here instead?

Reply:
This was never meant to i just took the first thing that worked, this was not meant to last. See #2337 and #2340 for the follow-ups

Reply:
Isn't it ok to use `--exclude-newer` here as long as it's this later date? I added that.